tests/mocha/blocks/procedures_test.js

```diff
@@ -1399,123 +1447,6 @@ suite('Procedures', function() {
         });
       });
 
-      suite('Enable/Disable', function() {
```

Comment:
Why was this test suite removed?

Reply:
I didn't realize this one existed, so I wrote a different one I like more in the previous PR hehe